tests/ssl-opt.sh

```diff
@@ -634,6 +635,15 @@ detect_dtls() {
     fi
 }
 
+# check if the given command uses dtls and sets global variable DTLS
+detect_gnutls_serv() {
+    if echo "$1" | grep 'gnutls-serv \|gnutls-serv$' >/dev/null; then
+        GNUTLS_SERV=1
```

Comment:
`GNUTLS_SERV` is also the path passed as an optional input in the initial environment. I\m not sure if using this name at this point breaks anything, but at best it's fragile.

Reply:
It does break something: it interferes with `requires_gnutls`.